src/UiDatepickerCalendar.vue

```diff
@@ -80,6 +80,8 @@ import UiCalendarMonth from './UiCalendarMonth.vue';
 import dateUtils from './helpers/date';
 import { scrollIntoView } from './helpers/element-scroll';
 
+const views = ['date', 'year'];
```

Comment:
`views` could be in-lined in the validator.

Reply:
Yeah, it could, but then you have the string "date" number of times in the minified code (I guess). Besides, it's like single source of truth and if you change it somewhere, you don't have to search and replace. I can still change it, it's your lib, your way :)